src/main/java/com/google/sps/data/ChallengeData.java

```diff
@@ -0,0 +1,34 @@
+
+package com.google.sps.data;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+public final class ChallengeData {
+
+    //Challenge names
+    private static final String GARDENING_CHALLENGE_0 = "Environmentally friendly fertilizer!";
+    //private static final String GARDENING_CHALLENGE_1 = "Incorporate native plants!";
+
+    private static final String RECYCLE_CHALLENGE_0 = "Old Electronics";
+    //private static final String RECYCLE_CHALLENGE_1 = "Donate";
+
+    private static final String WASTE_CHALLENGE_0 = "Create your own compost";
+    //private static final String WASTE_CHALLENGE_1 = "Avoid disposable goods";
+
+    //Challenge steps
+    private static final List GARDENING_STEPS_0 = Arrays.asList("step 1", "step 2", "step 3");
+    private static final List RECYCLE_STEPS_0 = Arrays.asList("step 1", "step 2", "step 3");
+    private static final List WASTE_STEPS_0 = Arrays.asList("step 1", "step 2", "step 3");
+
+    public static final Challenge[] challenges = {
```

Comment:
I'm not sure if you plan to access individual challenges in the future, but `ChallengeData.challenge[0]`, `ChallangeData.challenge[1]`, etc... is more difficult to read than something explicitly addressable like `ChallengeData.Gardening`, `ChallengeData.Recycle`, etc...

It seems right now your only use of ChallengeData is to access all the challenges together in the list form (in ChallengesServlet.convertToJsonUsingGson()), so keeping the only public member of ChallengeData in list form seems acceptable.

Reply:
In the future, I plan to separate the challenges based on their type.